### PR TITLE
wr-53: Implement author component on article page

### PR DESCRIPTION
**Pull request by kovAnya**

### CODE REVIEW

File: frontend/src/assets/css/scaffolding.scss [hunk 1]
```diff
@@ -32,3 +33,29 @@ body {
...
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
+  margin: 0;
+}
```
Comment (andanf-e) [inline frontend/src/assets/css/scaffolding.scss]: Margin is already reset for all elements upon `*` selector - add also `padding: 0` for all to reset
```suggestion
ul,
ol,
li {
  list-style: none;
}
```

**Review by andanf-e — CHANGES_REQUESTED** [review]

File: frontend/src/pages/article/article.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { type FC } from 'react';
+import { useParams } from 'react-router-dom';
+
+import { Author } from './components/author/author.js';
```
Comment (hnoievets) [inline frontend/src/pages/article/article.tsx:5]: @andanf-e  Shouldn't the `Author` component be imported from `~/libs/components/components.js`?

Comment (kovAnya) [reply frontend/src/pages/article/article.tsx:5]: I put it there because I suggest it is not a common component, but just a part of the article page

Comment (andanf-e) [reply frontend/src/pages/article/article.tsx:5]: Yep, @kovAnya is right - it's not a common component

File: frontend/src/pages/article/article.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { type FC } from 'react';
+import { useParams } from 'react-router-dom';
```
Comment (andanf-e) [inline frontend/src/pages/article/article.tsx]: Import it from hooks 
```suggestion
import { useParams } from '~/libs/hooks/hooks.js';
```

**Review by andanf-e — CHANGES_REQUESTED** [review]

File: frontend/src/pages/article/components/author/author.tsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+};
+
+const Author: FC<Properties> = ({
+  name = 'Charlie Culhane',
+  followers = 10,
+  rating = 700,
+  publishedAt = '2023-04-22',
+  readingTime = '7 min',
+  genre = 'Fiction',
+}) => {
```
Comment (NazarKovtun1) [inline frontend/src/pages/article/components/author/author.tsx:23]: Do not forget to remove these stubs when you work with real data

**Comment by b0bbobovich** [comment]
You can find some notes about your task here https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/70 :
https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/70#discussion_r1307206429
https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/70#discussion_r1307217651
https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/70#discussion_r1307218024

File: frontend/src/pages/article/components/author/author.tsx (hunk 4 above)
Comment (kovAnya) [reply frontend/src/pages/article/components/author/author.tsx:23]: thanks, got it

**Review by andanf-e — DISMISSED** [review]

**Review by andanf-e — APPROVED** [review]

**Review by NazarKovtun1 — APPROVED** [review]

**Pull request merged**